test/unit/lib/aws/awssigv4signer.test.js

```diff
@@ -50,6 +50,7 @@ test('Sign with SigV4', (t) => {
   const signedRequest = auth.buildSignedRequestObject(request);
   t.hasProp(signedRequest.headers, 'X-Amz-Date');
   t.hasProp(signedRequest.headers, 'Authorization');
+  t.hasProp(signedRequest.headers, 'x-amz-content-sha256');
```

Comment:
Is the value always the same? Maybe add a test that ensures this value is what you expect it to be?